### ISSUE TITLE
[Bug]: unable to join server due to Internal Exception

### ISSUE THREAD

**Issue by Nycto97** [Fuzss/horseexpert#8]
### Mod Loader (Required)

Forge

### Minecraft Version (Required)

1.19.2

### Mod Version (Required)

v4.0.0

### Notes (Required)

When trying to join a server (my own dedicated server, hosted on game-pc) while having Horse Expert and [Ender Chested](https://www.curseforge.com/minecraft/mc-mods/ender-chested) installed, along with my other mods, the following screen will appear:
![image](https://github.com/Fuzss/horseexpert/assets/83367226/12dfbaa0-989a-4c5d-8f32-4b1d8e23568b)

I'm running 400+ mods and this only occurs whenever Ender Chested is installed. Horse Expert will work along all other mods, as long as Ender Chested is not installed and vice versa.

I tested and running only these mods causes no issue:
![2023-10-25_03 44 29](https://github.com/Fuzss/horseexpert/assets/83367226/4d236c35-51fe-45e9-a083-d15ce875023f)

Client mods: 
[client-mod-list_2023-10-25_03.59.36.txt](https://github.com/Fuzss/horseexpert/files/13160587/client-mod-list_2023-10-25_03.59.36.txt)

Server mods: 
[server-mod-list_2023-10-25_04.06.48.txt](https://github.com/Fuzss/horseexpert/files/13160604/server-mod-list_2023-10-25_04.06.48.txt)

Sorry if i should report this on the Ender Chested repo instead, I have no clue at the moment tbh.

### latest.log (Optional)

https://gist.github.com/Nycto97/332e31eee428771d8f769cafdd3a3e98

**Comment by Nycto97** [comment]
Okay, scrap this report. Found out this Exception has to do with [Just Enough Beacons](https://www.curseforge.com/minecraft/mc-mods/just-enough-beacons-reforged) by @CerbonXD and the [Double Slabs](https://www.curseforge.com/minecraft/mc-mods/double-slabs) mod.

I reported a NullPointerException [here](https://github.com/CERBON-MODS/Just-Enough-Beacons/issues/2).

While the shown Exception in this post is different, they're definitely related, since removing Just Enough Beacons allows me to join my server, with all my mods, including Horse Expert, Ender Chested, I even just added Magnum Torch. Adding this last one and experiencing the same Exception again, made me realize the issue isn't related to Horse Expert or Ender Chested.

I'll leave this open for now, feel free to close it whenever.
